platform/barefoot/bfn-platform-ingrasys.mk

```diff
@@ -1,5 +1,7 @@
-BFN_INGRASYS_PLATFORM = bfnplatform-ingrasys_8.2.0_amd64.deb
-$(BFN_INGRASYS_PLATFORM)_URL = "https://github.com/Ingrasys-sonic/packages/raw/master/lib/bfnplatform-ingrasys_8.4.0_8.5.0_amd64.deb"
+ifdef BLDENV
```

Comment:
why doing this?

Reply:
1. Barefoot SDE already update to ver 8.8.1, the platform deb need to sync with SDE
2. Currently the new version only support stretch

Reply:
the master version is stretch.

Reply:
Do you mean the "ifdef BLDEVN" not needed here ? According to bfn-platform.mk, there are two different platform deb packages for stretch and jessie, but in our case only stretch is supported, so add the "ifdef BLDEVN" without "else".
Refer to https://github.com/Azure/sonic-buildimage/blob/master/platform/barefoot/bfn-platform.mk

Reply:
if the "ifdef BLDEVN" removed, the installation of the platform deb will fail due to dependency issue for jessie. Since our platform deb no longer support jessie build but only for stretch build, it should be ok not installing the deb for jessie build. Any better suggestion ?